semanticdb/semanticdb/semanticdb.proto

```diff
@@ -280,6 +280,7 @@ message SymbolInformation {
   Signature signature = 17;
   repeated Annotation annotations = 13;
   Access access = 18;
+  repeated string overriden_symbols = 19;
```

Comment:
Changes to semanticdb.proto need to be documented in three different places in the spec.

* Generic section on `SymbolInformation` https://scalameta.org/docs/semanticdb/specification.html#symbolinformation
* Scala section on `SymbolInformation`. We can link to the Scala Language Spec here https://www.scala-lang.org/files/archive/spec/2.12/05-classes-and-objects.html#overriding
* Java section on `SymbolInformation`. The relevant link to the Java Language spec https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.4.8

Reply:
Added documentation in all the needed places (I think)